mozregression/fetch_configs.py

```diff
@@ -68,6 +68,7 @@ class CommonConfig(object):
     """
 
     app_name = None
+    artifact_name = None
```

Comment:
I would prefer to see this in B2GDeviceConfigMixin I think (same level as inbound_branch=None).